test/conformance/gateway_conformance_test.go

```diff
@@ -63,35 +68,30 @@ func TestGatewayConformance(t *testing.T) {
 		t.Fatalf("unsupported KongRouterFlavor: %s", rf)
 	}
 
-	cSuite, err := suite.NewExperimentalConformanceTestSuite(
-		suite.ExperimentalConformanceOptions{
-			Options: suite.Options{
-				Client:               k8sClient,
-				GatewayClassName:     gatewayClassName,
-				Debug:                true,
-				CleanupBaseResources: !testenv.IsCI(),
-				BaseManifests:        conformanceTestsBaseManifests,
-				SupportedFeatures:    sets.New(supportedFeatures...),
-				SkipTests:            skippedTests,
-			},
-			ConformanceProfiles: sets.New(
-				suite.HTTPConformanceProfileName,
-			),
-			Implementation: conformancev1alpha1.Implementation{
-				Organization: metadata.Organization,
-				Project:      metadata.ProjectName,
-				URL:          metadata.ProjectURL,
-				Version:      metadata.Release,
-				Contact: []string{
-					path.Join(metadata.ProjectURL, "/issues/new/choose"),
-				},
-			},
-		},
+	opts := conformance.DefaultOptions(t)
```

Comment:
Instead of using `DefaultOptions` I think we should define the `Options` as we did before this change, since https://github.com/kubernetes-sigs/gateway-api/pull/2979 has been merged.

Reply:
https://github.com/Kong/kubernetes-ingress-controller/pull/5863#discussion_r1566172130

When I upgrade GWAPI to the latest version in my next PR, I will fix it. This PR aims to make reviews easier and let us know what we need to do before GRPCRoute GA.